tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -63,6 +64,37 @@ def get_cpu_template_dir(cpu_template):
     return cpu_template if cpu_template else "none"
 
 
+@pytest.mark.skipif(
+    PLATFORM != "aarch64",
+    reason="This is aarch64 specific test.",
+)
+def test_host_vs_guest_cpu_features_aarch64(uvm_nano):
+    """Check CPU features host vs guest"""
+
+    vm = uvm_nano
+    vm.add_net_iface()
+    vm.start()
+    host_feats = set(utils.check_output(CPU_FEATURES_CMD).stdout.strip().split(" "))
+    guest_feats = set(vm.ssh.check_output(CPU_FEATURES_CMD).stdout.strip().split(" "))
+
+    cpu_model = cpuid_utils.get_cpu_model_name()
+    match cpu_model:
```

Comment:
nit: I think using match here is overkill, but it could grow more complex in the future.

Reply:
I think match on an enum is better than if/else combos